kata-deploy/scripts/kata-deploy.sh

```diff
@@ -84,12 +88,26 @@ function configure_containerd() {
 	# https://github.com/kata-containers/packaging/issues/307
 	cat <<EOT | tee "$containerd_conf_file"
 [plugins]
-    [plugins.cri.containerd]
-      [plugins.cri.containerd.untrusted_workload_runtime]
-        runtime_type = "io.containerd.runtime.v1.linux"
-        runtime_engine = "/opt/kata/bin/kata-runtime"
-        runtime_root = ""
+  [plugins.cri]
+   [plugins.cri.containerd]
+     [plugins.cri.containerd.runtimes.kata]
+        runtime_type = "io.containerd.kata.v2"
 EOT
+
+	#Currently containerd has an assumption on the location of the shimv2 implementation
+	#Until support is added (see https://github.com/containerd/containerd/issues/3073), create
+	#a link to the v2-shim implementation at /opt/kata/bin at /usr/local/bin
```

Comment:
Nit: This might be slightly clearer worded as:

> create a link in /usr/local/bin/ to the v2-shim implementation in /opt/kata/bin.

Reply:
done